CautionaryAlertsApi/V1/UseCase/GetCautionaryAlertByAlertIdUseCase.cs

```diff
@@ -0,0 +1,26 @@
+using CautionaryAlertsApi.V1.Gateways;
+using Hackney.Shared.CautionaryAlerts.Boundary.Response;
+using System.Threading.Tasks;
+using System;
+using CautionaryAlertsApi.V1.UseCase.Interfaces;
+using Hackney.Shared.CautionaryAlerts.Factories;
+
+namespace CautionaryAlertsApi.V1.UseCase
+{
+    public class GetCautionaryAlertByAlertIdUseCase : IGetCautionaryAlertByAlertIdUseCase
+    {
+        private readonly IUhGateway _gateway;
+
+        public GetCautionaryAlertByAlertIdUseCase(IUhGateway gateway)
+        {
+            _gateway = gateway;
+        }
+
+        public CautionaryAlertResponse ExecuteAsync(Guid personId, Guid alertId)
+        {
+            var result = _gateway.GetCautionaryAlertByAlertId(personId, alertId);
+
+            return result?.ToResponse();
```

Comment:
_(Optional - for failure proofing the code)_
Need validation to check for `Guid.Empty`.

When there's a mistake in the Guid string, upon it getting parsed to Guid type, it gets set to Guid.Empty (a.k.a 0000000-0000-...)

And could possibly add some extra controller tests with 400 response.